packages/isc/sandbox_interface.go

```
@@ -301,6 +301,46 @@ type StateAnchor struct {
 	ISCPackage sui.Address
 }
 
+func NewStateAnchor(
+	ref *iscmove.AnchorWithRef,
+	owner *cryptolib.Address,
+	iscPackage sui.Address,
+) StateAnchor {
+	return StateAnchor{
+		Ref:        ref,
+		Owner:      owner,
+		ISCPackage: iscPackage,
+	}
+}
+
+func (s StateAnchor) GetObjectRef() *sui.ObjectRef {
+	return &s.Ref.ObjectRef
+}
+
+func (s StateAnchor) GetObjectID() *sui.ObjectID {
+	return s.Ref.ObjectID
+}
+
+func (s StateAnchor) GetAssets() *Assets {
+	// FIXME
+	return nil
```

Use `panic()` instead of `return nil` so that it's easier to find the error when it blows up in our face.

Also, as discussed IMO this method should not exist because we can't store all assets of the anchor in memory.